app/models/organisation.rb

```diff
@@ -82,13 +82,17 @@ def generic_team
 
   def generic_clinic
     locations.find_by(ods_code:, type: :generic_clinic) ||
-      Location.create!(
-        name: "Community clinics",
-        team: generic_team,
-        ods_code:,
-        type: :generic_clinic,
-        year_groups:
-      )
+      ActiveRecord::Base.transaction do
+        Location
+          .create!(
+            name: "Community clinics",
+            team: generic_team,
+            ods_code:,
+            type: :generic_clinic,
+            year_groups:
+          )
+          .tap { it.create_default_programme_year_groups!(programmes) }
+      end
```

Comment:
Just checking this is meant to be here, since it doesn't match the commit message. It looks right to me, just probably was missed from an earlier commit.